platform-common/src/test/java/com/softicar/platform/common/core/logging/LogFileOutputTest.java

```diff
@@ -0,0 +1,31 @@
+package com.softicar.platform.common.core.logging;
+
+import com.softicar.platform.common.testing.AbstractTest;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Test;
+
+public class LogFileOutputTest extends AbstractTest {
+
+	@Test
+	public void test() throws IOException {
+
+		// setup
+		Path tempFile = Files.createTempFile(getClass().getCanonicalName(), null);
+		tempFile.toFile().deleteOnExit();
+
+		// execute
+		try (var logFileOutput = new LogFileOutput(tempFile)) {
+			logFileOutput.logLine("foo");
+			logFileOutput.logLine("bar");
+		}
+
+		// assert
+		List<String> lines = Files.readAllLines(tempFile);
```

Comment:
somehow we should check that line termination characters matches the system default, e.g. for window '\r\n'...any ideas how? Maybe LogFileOuput should take the character in constructor with default `System.lineSeparator()`.

Reply:
`Files.readAllLines` uses `BufferedReader.readLine()` which supports all usual kinds of line separators.
And after all, this just a test. So what's the point?

Reply:
the point is that we should test that the `LogFileOuput` respects the `System.lineSeparator()`

Reply:
Obsolet because we now use `PrintWriter`.